server/lib/tuist_web/router.ex

```diff
@@ -36,7 +36,19 @@ defmodule TuistWeb.Router do
   end
 
   pipeline :browser_app do
-    plug :accepts, ["html", "svg", "png"]
+    plug :accepts, ["html"]
+    plug :disable_robot_indexing
+    plug :fetch_session
+    plug :fetch_live_flash
+    plug :put_root_layout, html: {TuistWeb.Layouts, :app}
+    plug :protect_from_forgery
+    plug :put_secure_browser_headers
+    plug :fetch_current_user
+    plug :content_security_policy
+  end
+
+  pipeline :browser_app_image do
```

Comment:
`put_root_layout` for an image probably doesn't make sense? but otherwise, looks good